slides/quickstart/index.html

```diff
@@ -61,40 +42,18 @@
        *  appropriately. After a sign-in, the API is called.
        */
       function updateSigninStatus(isSignedIn) {
-        if (isSignedIn) {
-          authorizeButton.style.display = 'none';
-          signoutButton.style.display = 'block';
-          listSlides();
-        } else {
-          authorizeButton.style.display = 'block';
-          signoutButton.style.display = 'none';
-        }
+        if (isSignedIn) listSlides();
```

Comment:
Style guide says you should always use braces:

https://google.github.io/styleguide/jsguide.html#formatting-braces-all

Reply:
The exception right below it states it can be omitted to improve readability.

```js
if (isSignedIn) {
  listSlides();
}
```
vs
```js
if (isSignedIn) listSlides();
```

`updateSigninStatus` seems pretty readable to me, but it is subject to debate.

Reply:
Yep, I should have read further down :-)